packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -121,7 +121,7 @@ export class FastSync extends (EventEmitter as {new (): InitialSyncEventEmitter}
   };
 
   private setBlockImportTarget = (fromSlot?: Slot): void => {
-    const lastTarget = fromSlot || this.blockImportTarget;
+    const lastTarget = typeof fromSlot === "number" ? fromSlot : this.blockImportTarget;
```

Comment:
I think you are looking for https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing :smile: